ospd_openvas/daemon.py

```diff
@@ -132,13 +132,20 @@
         ),
     },
     'optimize_test': {
-        'type': 'integer',
+        'type': 'boolean',
         'name': 'optimize_test',
-        'default': 5,
+        'default': 1,
         'mandatory': 0,
         'visible_for_client': True,
         'description': (
-            'By default, openvas does not trust the remote ' + 'host banners.'
+            'By default, openvas does not trust the remote host banners. '
```

Comment:
It's the other way around. optimize_test is 1/yes by default which means remote host banners and so on are trusted which can be disabled by setting optimize_test to 0/no.